### PR TITLE
RedM Discord Presence

### PR DESCRIPTION
**Pull request by Ktos93**
Require to set RedM app id and app asset

### CODE REVIEW

**Review by Slypher — APPROVED** [review]

**Review by vinicius7199 — APPROVED** [review]

**Comment by egerdnc** [comment]
Damn

**Review by sorambr — APPROVED** [review]

File: code/components/lovely-script/src/Lovely.cpp [hunk 1]
```diff
@@ -1,177 +1,173 @@
-/*
```
Comment (blattersturm) [inline code/components/lovely-script/src/Lovely.cpp:1]: Why does this remove and re-add the entire file?

**Review by blattersturm — CHANGES_REQUESTED** [review]
Does whoever 'approves these changes' actually check the changeset at all?

File: code/components/lovely-script/src/Lovely.cpp (hunk 1 above)
Comment (Ktos93) [reply code/components/lovely-script/src/Lovely.cpp:1]: Looks like the PR was broken due to the time it was waiting for, I'll do it again and it should look good

**Pull request closed without merging**